From corbet@lwn.net  Tue Jun  2 11:58:47 2009
From: Jonathan Corbet <corbet@lwn.net>
Date: Mon, 25 May 2009 10:15:27 -0600
Subject: debugfs: fix docbook error
To: akpm@linux-foundation.org
Cc: greg@kroah.com, shen@cn.fujitsu.com
Message-ID: <20090525101527.25163d2f@bike.lwn.net>


Fix an error in debugfs_create_blob's docbook description

It cannot actually be used to write a binary blob.

Signed-off-by: Jonathan Corbet <corbet@lwn.net>

---
 fs/debugfs/file.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/fs/debugfs/file.c
+++ b/fs/debugfs/file.c
@@ -419,7 +419,7 @@ static const struct file_operations fops
 };
 
 /**
- * debugfs_create_blob - create a debugfs file that is used to read and write a binary blob
+ * debugfs_create_blob - create a debugfs file that is used to read a binary blob
  * @name: a pointer to a string containing the name of the file to create.
  * @mode: the permission that the file should have
  * @parent: a pointer to the parent dentry for this file.  This should be a
